.github/workflows/close-stale.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Close stale issues and PRs'
+on:
+  pull_request: { }
+  push:
+    branches: [ feature/adm/stale-prs ]
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: sonia-corporation/stale@1.54.0
```

Comment:
uses a staleness action fork to allow for only checking PRs instead of PRs and issues which causes the rate limit to be hit due to our large amount of issues